.travis.yml

```diff
@@ -0,0 +1,24 @@
+# @Date:   2017-03-20T16:36:18-04:00
+# @Project: scatt-jacc-caj
+# @Filename: travis.yml
+# @Last modified time: 2017-03-20T16:53:05-04:00
+
+#defining the language used
+language: java
+
+script:
+#installing dependencies
+- ./gradlew assemble
+#calling default script
+- ./gradlew check
+
+
+#the following lines help travis avoid uploading of unneccessary caches related to

```

Comment:
I'm glad that you found these, but where did you get them? Is this from stack overflow? I'd like to see the explanation of the person who came up with them.